Scripts/RemediationScripts/Remediate-EnableHttpsForStorageAccount.ps1

```diff
@@ -0,0 +1,723 @@
+        3. Execute the script to enable "EnableHTTPSTrafficOnly" on the Storage Account in the Subscription. Refer `Examples`, below.
+   
+    To roll back:
+        1. Download the script.
+        2. Load the script in a PowerShell session. Refer https://aka.ms/AzTS-docs/RemediationscriptExcSteps to know more about loading the script.
+        3. Execute the script to disable "EnableHTTPSTrafficOnly"  on the Storage Account in the Subscription. Refer `Examples`, below.
+
+# Examples:
+    To remediate:
+        1. To review the Storage Account in a Subscription that will be remediated:
+           Enable-StorageEncryptionInTransit  -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000  -DryRun
+
+        2. To enable "EnableHTTPSTrafficOnly"  on the Storage Account in a Subscription:
+           Enable-StorageEncryptionInTransit  -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 
+       
+        3. To enable "EnableHTTPSTrafficOnly"  on the Storage Account in a Subscription, from a previously taken snapshot:
+           Enable-StorageEncryptionInTransit  -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -FilePath C:\Users\Documents\AzTS\Remediation\Subscriptions\00000000_xxxx_0000_xxxx_000000000000\20211013_0608\EnableSecureTransit\StorageWithDisableHTTPs.csv
+
+        To know more about the options supported by the remediation command, execute:
+        
+        Get-Help Enable-StorageEncryptionInTransit -Detailed
+   
+    To rollback: 
+        1. To disable "EnableHTTPSTrafficOnly"  on the Storage Account in a Subscription, from a previously taken snapshot:
+           Disable-StorageEncryptionInTransit -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000  -FilePath  C:\Users\Documents\AzTS\Remediation\Subscriptions\00000000_xxxx_0000_xxxx_000000000000\20211013_0608\EnableSecureTransit\StorageWithDisableHTTPs.csv
+
+        To know more about the options supported by the roll back command, execute:
+        
+        Get-Help Disable-StorageEncryptionInTransit -Detailed 
+
+#########################################>
+
+
+function Setup-Prerequisites
+{
+
+<#
+        .SYNOPSIS
+        Checks if the prerequisites are met, else, sets them up.
+
+        .DESCRIPTION
+        Checks if the prerequisites are met, else, sets them up.
+        Includes installing any required Azure modules.
+
+        .INPUTS
+        None. You cannot pipe objects to Setup-Prerequisites.
+
+        .OUTPUTS
+        None. Setup-Prerequisites does not return anything that can be piped and used as an input to another command.
+
+        .EXAMPLE
+        PS> Setup-Prerequisites
+
+        .LINK
+        None
+    #>  
+    
+    Write-Host "Required modules are: Az.Account, Az.Resources, Az.Storage" -ForegroundColor $([Constants]::MessageType.Info)
+    Write-Host "Checking for required modules..."
+    $availableModules = $(Get-Module -ListAvailable Az.Resources, Az.Accounts,Az.Storage)
+    
+    # Checking if 'Az.Accounts' module is available or not.
```

Comment:
Please replace lines 94-137 with something similar to - https://github.com/azsk/AzTS-docs/blob/dfc912021e4e032d613fcc580d49f9ea4e3b2638/Scripts/RemediationScripts/Remediate-DoNotUseManagementCertificates.ps1#L75-L86.